scripts/create-associations.js

```diff
@@ -78,15 +78,10 @@ function combineByName(lineItems, creatives) {
   console.log('got all creatives');
 
   lineItems.forEach(function(lineItem) {
-    associations[lineItem.name] = {
-      lineItemId: lineItem.id
-    };
-  });
-
-  creatives.forEach(function(creative) {
-    if (associations[creative.name]) {
-      associations[creative.name].creativeId = creative.id;
-    }
+    associations[lineItem.name] = []
+    creatives.forEach(function(creative) {
+      associations[lineItem.name].push({ lineItemId: lineItem.id, creativeId: creative.id });
+    });
```

Comment:
I think this changes the behavior slightly. The original approach was to match creatives and line items that had the same name, and create an association for them (10 line items and 10 creatives = 10 associations). This will instead make an association for every line item with every creative (10 line items and 10 creatives = 100 associations).

We've had a use case for both, so let me check to see what our adops team thinks is more common.